source/Octopus.Manager.Tentacle/Octopus.Manager.Tentacle.csproj

```diff
@@ -139,6 +144,9 @@
     <PackageReference Include="NuGet.Packaging.Core.Types" Version="3.6.0-octopus-58692" />
     <PackageReference Include="NuGet.Versioning" Version="3.6.0-octopus-58692" />
     <PackageReference Include="Portable.BouncyCastle" Version="1.9.0" />
+    <PackageReference Include="TaskScheduler" Version="2.7.2" />
+  </ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)' != 'net8.0-windows'">
```

Comment:
We're making sure these old system package references are no longer used with net8.0